python/bin/rubin-alert-sim

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+from streamsim import commands
```

Comment:
Do we want to put this in the `lsst` namespace? Something like `lsst.alert.streamsim`? My fingers are hard-wired to expect that, but I'm not sure if I can justify it.

Reply:
I thought not because that namespace seems to never use `setup.py` - it seems to be EUPS-land and I got scared of trying to interoperate with the rest of the LSST Python code. This seemed like a simple direct way forward.

I also don't expect users to import this; it's an internal module for the binary to use.